lua/entities/gmod_wire_coldetector.lua

```diff
@@ -0,0 +1,247 @@
+						bonehit=i
+						break
+					end
+				end
+				local newentpos=data.HitEntity:GetPos()
+				local newentang=data.HitEntity:GetAngles()
+				data.HitEntity:SetPos(ent2pos)
+				data.HitEntity:SetAngles(ent2ang)
+				ent:GetPhysicsObject():SetAngles(anglesxd)
+				ent:GetPhysicsObject():SetPos(posxd)
+				ent:SetAngles(anglesxd)
+				ent:SetPos(posxd)
+				//print("hleh")
+				//print(util.IsValidPhysicsObject( data.HitEntity, 0) && !self.NoPhysics)
+				if util.IsValidPhysicsObject( data.HitEntity, 0) && (!self.NoPhysics || data.HitEntity:IsWorld()) then
+					self.Stick = constraint.Weld(ent, data.HitEntity,0,bonehit,0,true)
+					if data.HitEntity:IsWorld() then
+						//print("?")
+						ent:GetPhysicsObject():EnableMotion(false)
+						ent:GetPhysicsObject():Sleep()
+						//bonehit=0
+					end
+				else
+					ent:PhysicsDestroy()
+					ent:SetSolid(SOLID_NONE)
+					ent:FollowBone(data.HitEntity,bonehit)
+					
+				end
+				//print(trace.HitBox)
+				data.HitEntity:SetPos(newentpos)
+				data.HitEntity:SetAngles(newentang)
+			end
+		end)
+	end
+	self.Collided=true
+	self.CurrData=data
+	WireLib.TriggerOutput(self, "Collided", 1)
+	WireLib.TriggerOutput(self, "Delta Time", data.DeltaTime)
+	WireLib.TriggerOutput(self, "Hit Entity", data.HitEntity)
+	WireLib.TriggerOutput(self, "Hit Position", data.HitPos)
+	WireLib.TriggerOutput(self, "Velocity", data.OurOldVelocity)
+	WireLib.TriggerOutput(self, "Hit Entity Velocity", data.TheirOldVelocity)
+	WireLib.TriggerOutput(self, "Speed", data.Speed)
+	WireLib.TriggerOutput(self, "Hit Normal", data.HitNormal)
+	WireLib.TriggerOutput(self, "Entity Position", posxd)
+	WireLib.TriggerOutput(self, "Entity Angle", anglesxd)
+	WireLib.TriggerOutput(self, "Hit Entity Position", ent2pos)
+	WireLib.TriggerOutput(self, "Hit Entity Angle", ent2ang)
+	self:ShowOutput()
+end
+
+function ENT:RemoveLink()
+	self.StickTo=nil
+	if IsValid(self.Stick) then
+		self.Stick:Remove()
+	end
+	if IsValid(self.LinkedEnt) then
+		if IsValid(self.LinkedEnt:GetParent()) then
+			self.LinkedEnt:SetParent(nil)
+			self.LinkedEnt:PhysicsInit(SOLID_VPHYSICS)
+			self.LinkedEnt:SetSolid( SOLID_VPHYSICS )
+			timer.Simple(0,function() 
+				if IsValid(self.LinkedEnt) then
+					self.LinkedEnt:GetPhysicsObject():Wake() 
+				end
+			end)
+			
+		end
+	end
+end
+
+function ENT:TriggerInput( name, value )
+	if name == "Stick to world" then
+		self.StickToWorld=value
+		//print(value==0 && self.StickTo)
+		if value==0 && self.StickTo && self.StickTo:IsWorld() then
+			self:RemoveLink()
+		end
+	elseif name == "Stick to props" then
+		self.StickToProps=value
+		if value==0 && self.StickTo && !self.StickTo:IsNPC() && !self.StickTo:IsPlayer() && !self.StickTo:IsWorld() then
+			self:RemoveLink()
+		end
+	elseif name == "Stick to npc/player" then
+		self.StickToNPC=value
+		if value==0 && self.StickTo && (self.StickTo:IsNPC() || self.StickTo:IsPlayer()) then
+			self:RemoveLink()
+		end
+	end
+end
+/*function ENT:OnRemove( )
```

Comment:
Turn this into a standard lua comment (--[[...]]) or remove it entirely.
Also, put the comment signs on separate lines so removing them would be a single-line change.